packages/schedule/src/ScheduleContainer.spec.tsx

```diff
@@ -26,13 +26,15 @@ describe('ScheduleContainer', () => {
   );
 
   beforeEach(() => {
-    jest.spyOn(window, 'requestAnimationFrame').mockImplementation(cb => {
-      setTimeout(() => {
+    jest.spyOn(window, 'requestAnimationFrame').mockImplementation((cb: Function) => {
```

Comment:
[nit] Not sure if this would be more semantically correct, but doesn't this work with `void`?

Reply:
> doesn't this work with void?

I'm not 100% sure what that means, but I'm down to try it 👍.  Could you clarify what "this" refers to? Are you referring to the `(cb: Function)`?

Reply:
`Function` in TypeScript is just that, a function. But, we are using `cb()` as a `void` which is a more specific type.

`void` in this case is just saying that `cb` is a function which is returning no value.

Reply:
Thanks for clarifying, what do you think about the following?

```ts
beforeEach(() => {
  jest.spyOn(window, 'requestAnimationFrame').mockImplementation((cb: (arg?: any) => void) => {
    const timerId = setTimeout(() => {
      cb();
    }, 1000 / 60);

    return Number(timerId);
  });
  jest.spyOn(window, 'cancelAnimationFrame');
  (clearTimeout as jest.Mock).mockClear();
});
```

Reply:
Paired on this and we ended up typing the `cb` as `FrameRequestCallback` 👍 .